README.md

```diff
@@ -1597,8 +1597,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 | [Back To Top](#contents) |
 
 - ### **S**
-
-  - [Saadat Badgujar](https://github.com/Saadat-B)
```

Comment:
Please add this name back and your name in alphabetical order.
Let us know if you need any help doing that. 🙂

Reply:
@Panquesito7 hey, have a look solved what you have asked for :)